src/SerilogWeb.Classic/Classic/ApplicationLifecycleModule.cs

```diff
@@ -128,7 +129,7 @@ static void LogRequest(object sender, EventArgs e)
         {
             if (!_isEnabled) return;
 
-            var request = HttpContext.Current.Request;
+            var request = HttpContextCurrent.Request;
             Logger.Write(_requestLoggingLevel, "HTTP {Method} for {RawUrl}", request.HttpMethod, request.RawUrl);
```

Comment:
After the change, this is going to lead to a `NullReferenceException` for `request.HttpMethod` etc., correct? Perhaps with the benefit of hindsight, going for the uglier (but more obviously null-returning) `TryGetCurrentRequest()` is a safer option?